### PR TITLE
Minor grammatical changed to README

### PR DESCRIPTION
**Pull request by jmccartie**
Small changes to make the README more ... readable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,19 +22,19 @@ gem 'sprockets', '~> 3.0'
 
 ## Using Sprockets
 
-For most people interested in using Sprockets you will want to see [End User Asset Generation](guides/end_user_asset_generation.md) guide. This contains information about sprocket's directive syntax, and default processing behavior.
+For most people interested in using Sprockets, you will want to see [End User Asset Generation](guides/end_user_asset_generation.md) guide. This contains information about sprocket's directive syntax and default processing behavior.
```
Comment (maclover7) [inline README.md]: while we're here, can you change `sprocket's` to `Sprockets'`

Comment (jmccartie) [inline README.md]: You bet. one sec

File: README.md [hunk 2]
```diff
@@ -22,19 +22,19 @@ gem 'sprockets', '~> 3.0'
...
 
 ## Behavior
 
 ### Index files are proxies for folders
 
-In Sprockets index files such as `index.js` or `index.css` files inside of a folder will generate a file with the folder's name. So if you have a `foo/index.js` file it will compile down to `foo.js`. This is similar to Node.js's behavior of using [folders as modules](https://nodejs.org/api/modules.html#modules_folders_as_modules). It is also somewhat similar to the way that a file in `public/my_folder/index.html` can be reached by a request to `/my_folder`. This means that you cannot directly use an index file. For example this would not work:
+In Sprockets, index files such as `index.js` or `index.css` inside of a folder will generate a file with the folder's name. So if you have a `foo/index.js` file it will compile down to `foo.js`. This is similar to Node.js's behavior of using [folders as modules](https://nodejs.org/api/modules.html#modules_folders_as_modules). It is also somewhat similar to the way that a file in `public/my_folder/index.html` can be reached by a request to `/my_folder`. This means that you cannot directly use an index file. For example this would not work:
```
Comment (maclover7) [inline README.md]: So if you have a file named `foo/index.js`, it will compile down to `foo.js`

**Comment by maclover7** [comment]
Few other small grammar comments, while we are editing this file. :)

**Comment by jmccartie** [comment]
@maclover7 Done. Thanks!

**Comment by maclover7** [comment]
Can you squash down, and add `[ci skip]` to your commit?

**Pull request merged**